Sources/Extensions/Watch/ExtensionDelegate.swift

```diff
@@ -28,16 +28,11 @@ class ExtensionDelegate: NSObject, WKExtensionDelegate {
 
         UNUserNotificationCenter.current().delegate = self
 
-        var opts: UNAuthorizationOptions = [.alert, .badge, .sound, .criticalAlert, .providesAppNotificationSettings]
-        if #available(watchOS 6.0, *) {
-            opts.insert(.announcement)
-        }
```

Comment:
Removed, as it is now always included - as per the documentation.

<img width="1075" alt="Screenshot 2024-03-04 at 6 44 34" src="https://github.com/home-assistant/iOS/assets/35694712/38daef32-ff53-4a36-a51e-b9fca5cebd6f">